app/src/main/java/com/example/recipeapp/activities/RegisterActivity.java

```diff
@@ -129,8 +129,9 @@ public File resizeFile(final Bitmap image) {
         return resizedFile;
     }
 
+    @Nullable
     @SuppressLint("Range")
-    public String getFileName(final Uri uri) {
+    public String getFileName(final @NonNull Uri uri) {
```

Comment:
Consistency of ordering seems a touch different from other arguments.... Which do you prefer to go first? Nullability or final? A fair argument could be made for either.